config/logstash.yml

```diff
@@ -299,8 +299,8 @@
 #
 # ------------ Other Settings --------------
 #
-# Run Logstash with superuser (default: ALLOW)
-# on_superuser: BLOCK, ALLOW
+# Allow or block running Logstash as superuser (default: ALLOW)
+# run_as_superuser: BLOCK, ALLOW
```

Comment:
The language we selected in the initial PR was the config setting `on_superuser` (which conveys the setting's reactionary nature), because `run_as_superuser` _feels like_ an imperative. This is somewhat mitigated by the only options being `BLOCK` and `ALLOW`, and less confusing than the original `true`/`false`

Reply:
`allow_superuser` is finalized through the discussion. Let me plz know if we can still improve. Thanks.